### PR TITLE
feat: add crvSUSD and crvLINK v1 to migration

### PR DESCRIPTION
**Pull request by patcito**

### CODE REVIEW

**Comment by patcito** [comment]
- Found address [0xC25a3A3b969415c80451098fa907EC722572917F](https://etherscan.io/address/0xC25a3A3b969415c80451098fa907EC722572917F) in files app/containers/Vaults/migrationWhitelist.json. This address is from token `crvPlain3andSUSD`  
- Found address [0x5533ed0a3b83F70c3c4a1f69Ef5546D3D4713E44](https://etherscan.io/address/0x5533ed0a3b83F70c3c4a1f69Ef5546D3D4713E44) in files app/containers/Vaults/migrationWhitelist.json. This address is from vault `yvcrvPlain3andSUSD`  
- Found address [0x5a770DbD3Ee6bAF2802D29a901Ef11501C44797A](https://etherscan.io/address/0x5a770DbD3Ee6bAF2802D29a901Ef11501C44797A) in files app/containers/Vaults/migrationWhitelist.json. This address is from vault `yvCurve-sUSD`  
- Found address [0xcee60cFa923170e4f8204AE08B4fA6A3F5656F3a](https://etherscan.io/address/0xcee60cFa923170e4f8204AE08B4fA6A3F5656F3a) in files app/containers/Vaults/migrationWhitelist.json. This address is from token `linkCRV`  
- Found address [0x96Ea6AF74Af09522fCB4c28C269C26F59a31ced6](https://etherscan.io/address/0x96Ea6AF74Af09522fCB4c28C269C26F59a31ced6) in files app/containers/Vaults/migrationWhitelist.json. This address is from vault `yvlinkCRV`  
- Found address [0xf2db9a7c0ACd427A680D640F02d90f6186E71725](https://etherscan.io/address/0xf2db9a7c0ACd427A680D640F02d90f6186E71725) in files app/containers/Vaults/migrationWhitelist.json. This address is from vault `yvCurve-LINK`

File: app/containers/Vaults/migrationWhitelist.json [hunk 1]
```diff
@@ -158,5 +158,21 @@
...
     "apyTooltip": "Please migrate funds to our new vault to continue earning boosted yield.",
     "migrationMessage": "This vault is no longer boosted, but is still earning yield. Withdraw funds or migrate your balance to the new v2 Curve UST yVault to continue earning boosted yield."
+  },
+  {
+    "symbol": "crvSUSD",
+    "token": "0xC25a3A3b969415c80451098fa907EC722572917F",
+    "vaultFrom": "0x5533ed0a3b83F70c3c4a1f69Ef5546D3D4713E44",
+    "vaultTo": "0x5a770DbD3Ee6bAF2802D29a901Ef11501C44797A",
+    "apyTooltip": "Please migrate funds to our new vault to continue earning boosted yield.",
+    "migrationMessage": "This vault is no longer boosted, but is still earning yield. Withdraw funds or migrate your balance to the new v2 Curve SUSD yVault to continue earning boosted yield."
```
Comment (x48-crypto) [inline app/containers/Vaults/migrationWhitelist.json:168]: @fameal @dudesahn are these vaults still earning yield? I'm guessing no? @patcito maybe we should just use a generic migrate message (that is not scoped to a vault) in the future.

File: app/containers/Vaults/migrationWhitelist.json [hunk 2]
```diff
@@ -158,5 +158,21 @@
...
+  },
+  {
+    "symbol": "crvLINK",
+    "token": "0xcee60cFa923170e4f8204AE08B4fA6A3F5656F3a",
```
Comment (x48-crypto) [inline app/containers/Vaults/migrationWhitelist.json:172]: LGTM but waiting for @dudesahn to approve addresses

**Review by x48-crypto — APPROVED** [review]

File: app/containers/Vaults/migrationWhitelist.json (hunk 2 above)
Comment (patcito) [reply app/containers/Vaults/migrationWhitelist.json:172]: This is how I got the token address (you can run it in inspector):

```js
const resp = await fetch("https://vaults.finance/all")
const vaults = await resp.json()
vaults.forEach(v=>{if(v.address==="0x96Ea6AF74Af09522fCB4c28C269C26F59a31ced6")console.log(v.token.address)})
vaults.forEach(v=>{if(v.address==="0x5533ed0a3b83F70c3c4a1f69Ef5546D3D4713E44")console.log(v.token.address)})
```

File: app/containers/Vaults/migrationWhitelist.json (hunk 1 above)
Comment (fameal) [reply app/containers/Vaults/migrationWhitelist.json:168]: They will be earning for 2 more weeks

File: app/containers/Vaults/migrationWhitelist.json (hunk 2 above)
Comment (dudesahn) [reply app/containers/Vaults/migrationWhitelist.json:172]: tool is pretty cool, nice work

**Comment by dudesahn** [comment]
Updated the custom order json to also contain the new vaults, looks good to me

**Review by dudesahn — APPROVED** [review]

**Pull request merged**